webhooks.md

```diff
@@ -0,0 +1,160 @@
+# PayrollHero Webhooks
+
+PayrollHero has a webhook system available to notify about various events in an employee's lifecycle.
+
+## Data Objects
+
+The Webhook Interface makes use of standardized payload objects to make sure that the responses are easily handled.
+
+These are:
+
+### Employee Object
+
+```json
+{
+    "id": 1,
+    "account_id": 2,
```

Comment:
Confusing.  You are otherwise using an "account" object in the other objects.  I'm happy with ```account_id``` but consistency is needed.